infrastructure/validators/offenses-slashes.md

```diff
@@ -0,0 +1,232 @@
+confuse the network with conflicting statements and when found out this will be a BEEFY/GRANDPA
+Equivocation Offense.
+
+Equivocations usually occur when duplicate signing keys reside on the validator host. If keys are
+never duplicated, the probability of an honest equivocation slash decreases to near 0.
+
+## Punishments
+
+On Polkadot, offenses to the network can be punished depending on their severity. There are three main punishments: slashing, disabling, and reputation changes.
+
+### Slashing
+
+**Slashing** will happen if a validator misbehaves in the network. They and their nominators will
+get slashed by losing a percentage of their staked DOT/KSM, from as little as 0.01% up to 100%.
+
+Any slashed DOT/KSM will be added to the [Treasury](https://wiki.polkadot.network/docs/learn-polkadot-opengov-treasury){target=\_blank}. The rationale for this (rather than burning or distributing them as rewards) is that slashes may be reverted by simply paying out from the Treasury. This would
+be useful in situations such as faulty slashes. In the case of legitimate slashing, tokens are moved
+away from malicious validators to those building the ecosystem through the normal Treasury process.
+
+Slashing only occurs for active validations for a given nominator, and slashes are not mitigated by
+having other inactive or waiting nominations. They are also not mitigated by the validator operator
+running separate nodes; each node is considered its own entity for slashing purposes.
+
+!!!info "Multiple Active Nominations"
+  In rare instances, with very large bonds, a nominator may actively nominate several validators in a
+  single era. In this case, the slash is proportionate to the amount staked to that specific
+  validator. Note that you cannot control the percentage of stake allocated to each validator or
+  choose who your active validator will be (except in the trivial case of nominating a single
+  validator). Staking allocations are controlled by the [Phragmén algorithm](https://wiki.polkadot.network/docs/learn-phragmen#understanding-phragm%C3%A9n){target=\_blank}.
+
+
+Once a validator gets slashed, it goes into the state as an "unapplied slash". You can check this
+via
+[Polkadot-JS UI](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/staking/slashes){target=\_blank}.
+The UI shows it per validator, followed by all the affected nominators and the amounts. While
+unapplied, a governance proposal can be made to reverse it during a 27-day grace period, after which
+the slashes are applied.
+
+A slash may occur under the circumstances below:
+
+1.  Equivocations – A slash of 0.01% is applied with as little as a single evocation. The slashed amount increases to 100% incrementally as more validators also equivocate
+2.  Disputes – This may result from a validator trying to represent the contents of a block falsely. Slashing penalties of 100% may apply
```

Comment:
```suggestion
1.  Equivocations – a slash of 0.01% is applied with as little as a single evocation. The slashed amount increases to 100% incrementally as more validators also equivocate
2.  Disputes – this may result from a validator trying to represent the contents of a block falsely. Slashing penalties of 100% may apply
```